packages/ember-data/tests/integration/adapter/build-url-mixin-test.js

```diff
@@ -147,7 +147,7 @@ test('buildURL - with camelized names', function() {
   adapter.setProperties({
     pathForType: function(type) {
       var decamelized = Ember.String.decamelize(type);
-      return Ember.String.pluralize(decamelized);
+      return Ember.String.underscore(Ember.String.pluralize(decamelized));
```

Comment:
No, the default adapter has no normalization, but the tests expected underscored strings.